.markdownlintrc

```diff
@@ -1,10 +1,13 @@
 {
   "default": true,
-  "MD003": { "style": "atx" },
+  "MD002": false,
   "MD007": { "indent": 4 },
-  "MD013": { "line_length": 200 },
+  "MD013": { "line_length": 500 },
```

Comment:
This is here because of our custom tooltips, I'm not sure how to approach this one other than setting it super long.